st2common/st2common/hooks.py

```diff
@@ -30,6 +31,18 @@
 
 LOG = logging.getLogger(__name__)
 
+# A list of method names for which we don't want to log the result / response
+RESPONSE_LOGGING_METHOD_NAME_BLACKLIST = [
+    'get_all'
+]
+
+# A list of controller classes for which we don't want to log the result / response
+RESPONSE_LOGGING_CONTROLLER_NAME_BLACKLIST = [
```

Comment:
This is weird. A blacklist of controllers that exist in st2api having literals used in st2api.